kubeless-rbac.jsonnet

```diff
@@ -5,11 +5,33 @@ local objectMeta = k.core.v1.objectMeta;
 
 local kubeless = import "kubeless.jsonnet";
 local controller_account = kubeless.controller_account;
-local controller_roles = [{
-  apiGroups: ["*"],
-  resources: ["services", "deployments", "functions", "configmaps", "pods", "replicasets"],
-  verbs: ["*"]
-}];
+local controller_roles = [
+  {
+    apiGroups: [""],
+    resources: ["services", "configmaps"],
+    verbs: ["create", "get", "delete", "list", "update", "patch"],
+  },
+  {
+    apiGroups: ["apps", "extensions"],
+    resources: ["deployments"],
+    verbs: ["create", "get", "delete", "list", "update", "patch"],
+  },
+  {
+    apiGroups: ["extensions"],
+    resources: ["replicasets"],
```

Comment:
AFAIK replicasets are created by the deployments and not by the controller itself. Maybe it is not necessary to give permissions to create them?

Reply:
+1, removed `replicasets` rule